common/utils/budget.js

```diff
@@ -930,7 +1005,7 @@ export const sumShareCostsForFundingSource = ({
   totalMedicaidCostShares,
   costCategoryShare
 } = {}) => {
-  const updatedBudget = deepCopy(budget);
+  let updatedBudget = deepCopy(budget);
```

Comment:
I think this can stay a const because the values in the object can change but it can't change from an `object` to another type. Not 100% since it's being json stringified'd + parsed.

Reply:
Good thought! However, because towards the bottom of the function we're reassigning the `updatedBudget` variable to a new value, it only works with a `let` 
It's this line:
```
updatedBudget = sumShareCostsForCombinedTotals({
        budget: updatedBudget,
        year,
        medicaidShare: totalMedicaidCost,
        fedShare: totalMedicaidCostShares.fedShare,
        stateShare: totalMedicaidCostShares.stateShare
      });
```